orchestrator-salt/src/main/resources/salt/salt/gateway/config/cm/topology.xml.j2

```diff
@@ -261,12 +261,6 @@
                  <value>enabled=true;maxFailoverAttempts=3;failoverSleep=1000</value>
              </param>
              {%- endif %}
-             {% if 'WEBHBASE' in exposed and 'HBASERESTSERVER' in salt['pillar.get']('gateway:location') -%}
```

Comment:
is it ok if we remove this?

Reply:
Peter and I had talked about this -- we're not really sure why it was here in the first place.

The HBASERESTSERVER doesn't provide a UI. It's meant for programmatic REST access. By that stance, there's no good reason we can think of as to why it would be in the topology.xml.j2 file in addition to the topology-api.xml.j2

Reply:
Not sure is @risdenk happens to know?

Reply:
I don't know for sure. The history looks like it was added in https://github.com/hortonworks/cloudbreak/commit/f22a5279c4d916d73067b24a9800f4df88c89c3d. I don't know of any usage of this endpoint from the UI topology.

btw if this goes can also remove the `webhbase.acl` section a little further up.

Reply:
Nice catch @risdenk ! Removed that parameter too.